build-logic/build.gradle.kts

```diff
@@ -29,6 +29,7 @@ dependencies {
   implementation(libs.android.plugin)
   implementation(libs.gradle.japicmp.plugin)
   implementation(libs.vespene)
+  implementation(libs.kotlinx.serialization.json)
```

Comment:
Required to fix a classloader issue in Gradle when not relocating the plugin (which I do during local dev to save some time)